src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -2829,6 +2829,51 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
         enum E<const C: i32> {}
     """)
 
+    @MockRustcVersion("1.47.0")
+    fun `test min const generics E0658 1`() = checkErrors("""
+        fn f<<error descr="const generics is experimental [E0658]">const C: i32</error>>() {}
+        struct S<<error descr="const generics is experimental [E0658]">const C: i32</error>>(A);
+        trait T<<error descr="const generics is experimental [E0658]">const C: i32</error>> {}
+        enum E<<error descr="const generics is experimental [E0658]">const C: i32</error>> {}
+    """)
+
+    @MockRustcVersion("1.47.0-nightly")
+    fun `test min const generics E0658 2`() = checkErrors("""
+        #![feature(min_const_generics)]
+        fn f<const C: i32>() {}
+        struct S<const C: i32>(A);
+        trait T<const C: i32> {}
+        enum E<const C: i32> {}
+    """)
```

Comment:
This test fails now because `since` version for `min_const_generics` was changed after updating compiler features. I suggest not trying to support such false positives for old nightly toolchain. It seems that a proper solution requires retrieving compiler features from the actual compiler that's not possible for now.
Let's just check that for `1.51.0-nightly` we don't show error annotation either with `min_const_generics ` feature or without it